### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
+RUN mv bazel.gpg /etc/apt/trusted.gpg.d/
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+RUN apt update && apt install -y bazel
+
+ADD . /mayhem-perf_data_converter
+WORKDIR /mayhem-perf_data_converter
+RUN bazel build src:perf_to_profile
+RUN cp /mayhem-perf_data_converter/bazel-bin/src/perf_to_profile /
```
Comment (xansec) [inline Dockerfile:16]: Perhaps you meant for this to be done as a package stage? Are you able to split this into multiple stages in the Dockerfile to reduce the image size?

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+project: rnshah9/mayhem-perf-data-converter
+target: perf-to-profile
+
+cmds:
+  - cmd: /perf_to_profile -i @@ -o output
```
Comment (xansec) [inline Mayhemfile]: You can set output to `/dev/null` to prevent unnecessary filesystem writing

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+project: rnshah9/mayhem-perf-data-converter
```
Comment (xansec) [inline Mayhemfile]: you may also want to genericize this project to `project: perf-data-converter`

**Review by xansec — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (rnshah9) [reply Dockerfile:16]: Ah, yes I was able to. In general, is doing so a requirement?

Comment (xansec) [reply Dockerfile:16]: It's not a hard requirement, but it's better if you can implement multi-stage builds. We'll eventually have at least 1000 jobs running under mayhemheroes, and pushing and pulling that many images can take a while; keeping image size down is a best practice in general, but also for our purposes.

**Comment by xansec** [comment]
LGTM!

**Comment by xansec** [comment]
LGTM!

File: Dockerfile (hunk 1 above)
Comment (rnshah9) [reply Dockerfile:16]: Sounds good, I'll try to do so for all my submissions; please forgive me if I run behind your initial PR reviews!

**Pull request merged**